server/src/controllers/forests/firewood-permits.es6

```diff
@@ -0,0 +1,189 @@
+/* eslint-disable consistent-return */
+/* eslint no-param-reassign: ["error", { "props": false }] */
+/**
+ * @module controllers/forests/firewood-permits
+ */
+const logger = require('../../services/logger.es6');
+const firewoodModel = require('../../models/firewood-permits.es6');
+const forestsDb = require('../../models/forests.es6');
+// const permitSvgService = require('../../services/christmas-trees-permit-svg-util.es6');
```

Comment:
Can we remove this?

Reply:
Id vote to keep it for visibility in creating the firewood permit